quilt/tools/store.py

```diff
@@ -396,13 +396,12 @@ def save_df(self, df, name, path, ext, target):
         """
         self._find_path_write()
         buildfile = name.lstrip('/').replace('/', '.')
-        storepath = os.path.join(self._pkg_dir, buildfile)
+        storepath = self._object_path('.' + buildfile)
```

Comment:
Oh, that makes sense. It seems cleaner to keep the temporary files in another directory, but it's nice to be able to reuse _object_path. Although it's unlikely to come up very often, we should try to make those temporary files easy to locate and delete in case something goes wrong. The object directory gets big very fast.